packages/ai-openrouter/tests/openrouter-responses-adapter.test.ts

```
@@ -3,6 +3,7 @@ import { EventType, chat } from '@tanstack/ai'
 import { resolveDebugOption } from '@tanstack/ai/adapter-internals'
 import { ResponsesRequest$outboundSchema } from '@openrouter/sdk/models'
 import { createOpenRouterResponsesText } from '../src/adapters/responses-text'
+import type { OpenRouterResponsesTextProviderOptions } from '../src/adapters/responses-text'
```

ESLint

[error] `../src/adapters/responses-text` type import should occur after import of `../src/tools/web-fetch-tool`

(import/order)